packages/documentation/src/pages/getting-started/common-tasks/routing.mdx

```diff
@@ -0,0 +1,43 @@
+# Routing
+
+## Proxy Config
```

Comment:
```suggestion
## Reverse Proxy Config
```

VA.gov has a reverse proxy for web traffic and a forward proxy VA service traffic.